### PR TITLE
Fixes in section-02.ipynb for Python 3.6

### PR DESCRIPTION
**Pull request by alvaroing12**
Hi Parag, 

I have fixed the code in the lecture-02 IPhyton notebook.
1) In the Learning Rate section. 
This changes cast the init_p variable as an integer in order for it to be accepted in Python 3.6 as a viable index for the gradient, x and cost arrays. 
This issue was mentioned in the class forum [here](https://www.kadenze.com/forums/creative-applications-of-deep-learning-with-tensorflow-sessions-session-2-training-a-network-w-tensorflow/threads/index-error-when-iterating-through-n_iterations-in-2-6)

2) In the Over vs. Underfitting section
Fixed the power range in section to start at value 1. This avoids having a 0th power expansion which introduces additional bias. This issue was raised in the session 02 forum [here](https://www.kadenze.com/forums/creative-applications-of-deep-learning-with-tensorflow-sessions-session-2-training-a-network-w-tensorflow/threads/lecture-2-doubt-in-code-where-we-add-bias-in-over-vs-underfitting-section)

### CODE REVIEW

**Comment by pkmital** [comment]
Thank you!
